src/app/components/scenario-wizard/scenario-wizard.component.test.js

```diff
@@ -19,7 +19,7 @@ describe('ScenarioWizardComponent', () => {
 
     it('should emit event when criteria is removed', () => {
         sut.removeCriteria();
-        expect(sut.onRemoveCriteria.next).toHaveBeenCalled();
+        expect(sut.onRemoveCriteria.emit).toHaveBeenCalled();
```

Comment:
It would be better, if in next time when function called with parameter - check this calling, ex: 
```sh
sut.removeCriteria(param);
expect(sut.onRemoveCriteria.next).toHaveBeenCalledWith(param);
```